Sources/App/Utilities/SemverVersion.swift

```diff
@@ -1,14 +1,33 @@
 struct SemverVersion {
     enum Bump {
+        case stage
```

Comment:
Maybe call this `release` or `releaseStage`? That way it's more clear it relates to the `release` label.